gl_generator/generators.rs

```diff
@@ -117,3 +124,39 @@ pub fn gen_symbol_name(api: Api, cmd: &str) -> String {
         Api::Egl => format!("egl{}", cmd),
     }
 }
+
+/// Writes all types into their own sub-module.
+pub fn write_type_aliases<W>(registry: &Registry, dest: &mut W) -> io::Result<()>
+where
+    W: io::Write,
```

Comment:
If we're going with Rust 2018, perhaps we should make these `impl io::Write` now?

Reply:
We can probably just go full `dyn Write`